lib/plugin-webpack5.js

```diff
@@ -1,210 +1,218 @@
-const qs = require('querystring')
-const id = 'vue-loader-plugin'
-const NS = 'vue-loader'
-const BasicEffectRulePlugin = require('webpack/lib/rules/BasicEffectRulePlugin')
-const BasicMatcherRulePlugin = require('webpack/lib/rules/BasicMatcherRulePlugin')
-const DescriptionDataMatcherRulePlugin = require('webpack/lib/rules/DescriptionDataMatcherRulePlugin')
-const RuleSetCompiler = require('webpack/lib/rules/RuleSetCompiler')
-const UseEffectRulePlugin = require('webpack/lib/rules/UseEffectRulePlugin')
+const qs = require("querystring");
+const id = "vue-loader-plugin";
+const NS = "vue-loader";
+const BasicEffectRulePlugin = require("webpack/lib/rules/BasicEffectRulePlugin");
+const BasicMatcherRulePlugin = require("webpack/lib/rules/BasicMatcherRulePlugin");
+const DescriptionDataMatcherRulePlugin = require("webpack/lib/rules/DescriptionDataMatcherRulePlugin");
+const RuleSetCompiler = require("webpack/lib/rules/RuleSetCompiler");
+const UseEffectRulePlugin = require("webpack/lib/rules/UseEffectRulePlugin");
 
 const ruleSetCompiler = new RuleSetCompiler([
-  new BasicMatcherRulePlugin('test', 'resource'),
-  new BasicMatcherRulePlugin('mimetype'),
-  new BasicMatcherRulePlugin('dependency'),
-  new BasicMatcherRulePlugin('include', 'resource'),
-  new BasicMatcherRulePlugin('exclude', 'resource', true),
-  new BasicMatcherRulePlugin('conditions'),
-  new BasicMatcherRulePlugin('resource'),
-  new BasicMatcherRulePlugin('resourceQuery'),
-  new BasicMatcherRulePlugin('resourceFragment'),
-  new BasicMatcherRulePlugin('realResource'),
-  new BasicMatcherRulePlugin('issuer'),
-  new BasicMatcherRulePlugin('compiler'),
+  new BasicMatcherRulePlugin("test", "resource"),
+  new BasicMatcherRulePlugin("mimetype"),
+  new BasicMatcherRulePlugin("dependency"),
+  new BasicMatcherRulePlugin("include", "resource"),
+  new BasicMatcherRulePlugin("exclude", "resource", true),
+  new BasicMatcherRulePlugin("conditions"),
+  new BasicMatcherRulePlugin("resource"),
+  new BasicMatcherRulePlugin("resourceQuery"),
+  new BasicMatcherRulePlugin("resourceFragment"),
+  new BasicMatcherRulePlugin("realResource"),
+  new BasicMatcherRulePlugin("issuer"),
+  new BasicMatcherRulePlugin("compiler"),
   new DescriptionDataMatcherRulePlugin(),
-  new BasicEffectRulePlugin('type'),
-  new BasicEffectRulePlugin('sideEffects'),
-  new BasicEffectRulePlugin('parser'),
-  new BasicEffectRulePlugin('resolve'),
-  new BasicEffectRulePlugin('generator'),
-  new UseEffectRulePlugin()
-])
+  new BasicEffectRulePlugin("type"),
+  new BasicEffectRulePlugin("sideEffects"),
+  new BasicEffectRulePlugin("parser"),
+  new BasicEffectRulePlugin("resolve"),
+  new BasicEffectRulePlugin("generator"),
+  new UseEffectRulePlugin(),
+]);
 
 class VueLoaderPlugin {
-  apply (compiler) {
+  apply(compiler) {
+    const webpack = compiler.webpack.NormalModule || require("webpack/lib/NormalModule");
```

Comment:
Please revert the format changes.
And here the variable name is misleading. It's not a webpack instance, thus should not be named as `webpack`.